lib/rubocop/cop/lint/boolean_symbol.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for `:true` and `:false` symbols.
+      # In most cases it would be a typo.
+      #
+      # @example
+      #
+      #   # bad
+      #   :true
```

Comment:
While we are at it, we might as well add in the bad and good example for false. 2 extra lines of documentation that would then cover 100% of the cases.